src/coprocessor/endpoint.rs

```diff
@@ -84,17 +105,20 @@ impl Host {
             pool: ThreadPool::new(
                 thd_name!("endpoint-normal-pool"),
                 concurrency,
-                FifoQueue::new(),
+                DEFAULT_BATCH_SIZE,
```

Comment:
Pool will execute `on_tick` when handling every `DEFAULT_BATCH_SIZE` tasks or on timeout. Do you have a better name?